### ISSUE TITLE
Images of text should be replaced by text when possible

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#2745]
As per Trusted Tester for images of text, images of text can only be used if they cannot be replaced by actual text. 

On the [Revenues page](https://revenuedata.doi.gov/how-revenue-works/revenues), the Statute, Appropriation, Grant Disbursement image can be replaced by coded text and decorative icons.
![image.png](https://images.zenhubusercontent.com/63482c003bae1bf4351c6f26/fe7bc050-ddec-422b-9562-609aa79fc400)
